armsrc/appmain.c

```diff
@@ -440,7 +446,219 @@ void SamyRun()
 	}
 }
 #endif
+#endif
```

Comment:
The `#ifdef`-ing looks a bit overly complicated, wouldn't this work ?

    #ifdef WITH_ISO14443a_StandAlone
      ... new code ...
    #elif LF
     ... samy's LF-code ... 
    #endif